priam/src/main/java/com/netflix/priam/resources/PriamConfig.java

```diff
@@ -89,11 +92,14 @@ public Response getProperty(
             String value = priamServer.getConfiguration().getProperty(name, defaultValue);
             if (value != null) {
                 result.put(name, value);
-                return Response.ok(result, MediaType.APPLICATION_JSON).build();
+                return Response.ok(GsonJsonSerializer.getGson().toJson(result)).build();
             } else {
                 result.put("message", String.format("No such property: [%s]", name));
                 logger.error(String.format("No such property: [%s]", name));
-                return Response.status(404).entity(result).type(MediaType.APPLICATION_JSON).build();
+                return Response.status(404)
+                        .entity(GsonJsonSerializer.getGson().toJson(result))
+                        .type(MediaType.TEXT_PLAIN)
```

Comment:
Same as other.

Reply:
Fair point I have modified it to APPLICATION_JSON now.